rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
```

Comment:
- [ ] Please add the belong-to association to book. It can look something like this:
```
book.rentals << self
```